espnet/nets/pytorch_backend/rnn/decoders.py

```diff
@@ -232,45 +260,68 @@ def recognize_beam(self, h, lpz, recog_args, char_list, rnnlm=None, strm_idx=0):
         """beam search implementation
```

Comment:
Can you also update `ScorerInterface` methods (init_state, score) as same as this `recognize_beam`? We are moving into the interface at this time.

Reply:
Sure, thanks! I will do that